components/TranscriptionView.tsx

```diff
@@ -1,8 +1,12 @@
 import React from "react";
-import { View } from "react-native";
+import { View, Text} from "react-native";
 
-const TranscriptionView = (): JSX.Element => {
-	return <View></View>;
+const TranscriptionView = (prop:string): JSX.Element => {
```

Comment:
nit, would prefer a name that indicates the prop's purpose, like `transcription`

Reply:
actually, this doesn't look quite right

Reply:
it yelled at me that the implied type was 'any' so I put it to a string